common/render.d/all

```diff
@@ -28,8 +28,12 @@ render_mod_params ()
             channel=${CHARM_CHANNEL[$c]:-$service_default_channel}
             echo "-e '/__CHARM_CS_NS____CHARM_CH_PREFIX__${c}\s*$/a\ \ \ \ channel: $channel'" >> $config_renderer
 
-            prefix=${CH_PREFIXED_CHARMS[$c]:-""}
-            echo "-e 's,__CHARM_CS_NS____CHARM_CH_PREFIX__${c}\s*$,$prefix${c},'" >> $config_renderer
+            if `grep -q "__SKIP_PREFIX__$c" $template`; then
+                echo "-e 's,__CHARM_CS_NS____CHARM_CH_PREFIX____SKIP_PREFIX__${c}\s*$,${c},'" >> $config_renderer
```

Comment:
I am not familiar with if and when we might want to use the prefixed version of the `grafana` charm. I don't see how we would in case we wanted to.

Reply:
As far as I know the prefixed is only ever for use with osm.

Reply:
Ed, 

How about LMA for Charmed Kubernetes. Is operator based grafana (same as OSM one) used for Charmed kubernetes lma stack?

Reply:
@hemanthnakkina my understanding is that the osm prefixed charms are only for use with osm (they are developed and maintained separately to the others).

Reply:
Thanks for the update @dosaboy . Looks good to me then :)